features/step_definitions/networking.rb

```diff
@@ -1275,3 +1275,20 @@
   cb[cb_name]="172.31.249.80:9095"
   logger.info "The ipecho service url #{cb[cb_name]} is stored to the #{cb_name} clipboard."
 end
+
+Given /^I Collect multus related information/ do
+  ensure_admin_tagged
+  step %Q/I run the :get admin command with:/, table(%{
+    | resource | nodes |
+    | o        | wide  |
+  })
+  step %Q/I run the :describe admin command with:/, table(%{
+    | resource | daemonset        |
+    | n        | openshift-multus |
+  })
+  step %Q/I run the :get admin command with:/, table(%{
+    | resource | pods             |
+    | n        | openshift-multus |
+    | o        | wide             |   
+  })
+end
```

Comment:
Please add a newline at the end of this file so we don't see this symbol telling us no newline at the end of file: 
![image](https://user-images.githubusercontent.com/8180469/108758248-e1081400-7518-11eb-989b-6bb3de310494.png)